string event #Name of the event
geometry_msgs/PoseStamped meta_model #Systems model that needs to be re-established after handling